projectorganizer/src/prjorg-project.c

```diff
@@ -275,14 +275,15 @@ static GeanyFiletype *filetypes_detect(const gchar *utf8_filename)
 	return ft;
 }
 
-
 static void regenerate_tags(PrjOrgRoot *root, gpointer user_data)
 {
 	GHashTableIter iter;
 	gpointer key, value;
 	GPtrArray *source_files;
 	GHashTable *file_table;
+	gchar **open_files;
```

Comment:
I'd suggest renaming it to `session_files` because these aren't open yet and may be confusing.